src/include/denc.h

```diff
@@ -44,11 +44,12 @@
 #include "buffer.h"
 #include "byteorder.h"
 
-template<typename T, typename VVV=void>
+template<typename T, typename=void>
 struct denc_traits {
   static constexpr bool supported = false;
   static constexpr bool featured = false;
   static constexpr bool bounded = false;
+  static constexpr bool with_dencv = false;
```

Comment:
it's not clear to me what `dencv` means here. maybe something like `need_contiguous = true` would better express its purpose?